benchmarks/pvc/gemm_configuration.hpp

```diff
@@ -52,14 +52,17 @@ namespace cutlass {
 namespace gemm {
 namespace device {
 
+enum class Scheduler { Parallel, SplitK, StreamK };
```

Comment:
I think the basic algorithm is called data parallel. Technically all de some parallel processing. So is this worth renaming?
```suggestion
enum class Scheduler { DataParallel, SplitK, StreamK };
```

Reply:
After discussion with @aacostadiaz, the names have been changed more drastically